File: Iteration2/code/CSE3063GRP12/src/features/advisor/student_list/AdvisorStudentListView.java
package features.advisor.student_list;

import core.exceptions.UnexpectedInputException;
import core.models.concretes.Student;
import java.util.ArrayList;

public class AdvisorStudentListView{
    public void showStudentList(ArrayList<Student> studentArrayList){
        for(int i = 1;i<=studentArrayList.size();i++){
            System.out.println(i + ". " + studentArrayList.get(i-1).getUserName() + " " + studentArrayList.get(i-1).getLastName());
        }
    }
    public void showQuitMessage(){
        System.out.print("Press q to return Main Menu: ");
    }
    public void showErrorMessage() throws UnexpectedInputException{
        throw new UnexpectedInputException();
    }

}

File: Iteration3/code/CSE3063GRP12/src/features/advisor/student_list/AdvisorStudentListView.py
from core.exceptions.UnexpectedInputException import UnexpectedInputException

class AdvisorStudentListView:
    def showStudentList(self, studentArrayList):
        print("======================================================================================Student List======================================================================================")
        print("%-27s%-20s%-20s" % ("[Student ID]", "[First Name]", "[Last Name]"))
        print("========================================================================================================================================================================================")
        for i, student in enumerate(studentArrayList, start=1):
            print("[%d] %-20s %-20s %-20s" % (i, student.userName, student.firstName, student.lastName))
        print("========================================================================================================================================================================================")

    def showQuitMessage(self):
        print("Press q to return Main Menu: ")

    def showErrorMessage(self):
        raise UnexpectedInputException()
